core/services/relay/evm/functions/logpoller_wrapper_test.go

```diff
@@ -60,28 +55,42 @@ func newSubscriber(expectedCalls int) *subscriber {
 	return sub
 }
 
-func addr(t *testing.T, lastByte string) []byte {
```

Comment:
Given that passing `t` was not actually the issue which caused the race condition, can we revert this change?